package.json

```diff
@@ -119,15 +119,15 @@
     "puppeteer": "^1.16.0",
     "recursive-readdir": "^2.2.2",
     "request": "^2.88.0",
-    "rollup": "^1.1.2",
-    "rollup-plugin-babel": "^4.3.2",
+    "rollup": "^2.29.0",
+    "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-glob-import": "^0.4.5",
     "rollup-plugin-istanbul": "^2.0.1",
-    "rollup-plugin-jscc": "^1.0.0",
-    "rollup-plugin-license": "^0.12.1",
+    "rollup-plugin-jscc": "^2.0.0",
+    "rollup-plugin-license": "^2.2.0",
     "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-terser": "^5.1.2",
+    "rollup-plugin-terser": "^7.0.2",
```

Comment:
I needed to upgrade these dependencies in order to get the API I needed to write `scripts/watchFunctionalTests.js`.